querydsl-codegen/src/test/java/com/querydsl/codegen/EntitySerializerTest.java

```diff
@@ -78,7 +78,7 @@ public void No_Package() throws IOException {
     }
 
     @Test
-    public void OriginalCategory() throws IOException {
+    public void originalCategory() throws IOException {
```

Comment:
Don't you think this one stands out between all the `snake_case`?
If it were a class or property name, I'd say preserve the casing.
Maybe we should `snake_case` all 'test specifications'?

What do you think?